channel-manager-api.md

```diff
@@ -209,6 +201,111 @@ Mews automatically sends changes in Inventory \(once connection is set up\). Dat
 | `7` | Closed to departure |
 | `8` | Closed to stay |
 
+When all restrictions removed, state |`1`| is sent.
+
+```javascript
+       {
+           "ratePlanCode": "r",
+           "spaceTypeCode": "JST",
+           "state": [
+               1
+           ],
+           "minLos": null,
+           "maxLos": null,
+           "from": "2019-09-24",
+           "to": "2019-10-31"
+       }
+```
+
+New restrictions override old restrictions. State |`1`| is not sent to remove old restrictions, if they were replaced.
+
+State |`2`| is always sent in combination with state |`6`|, |`7`|, or |`8`|, or all together.
```

Comment:
Why would we send all 4 states together?

Reply:
we send all 4 states together, when rate is disabled in Commander. Shall I add this or this would be confusing for a third party and all they need to know that such combination exists?

Reply:
imho, it makes more sense to send only state 8 for a disabled rate. @patrikmiksovsky95 why do we send all states when a rate is disabled in Mews?

Reply:
@ondrahermanek ? 😄 why we did that ? I think we wanted to just stop all of it.

Reply:
ditch the `|` around the numbers, they are used for tables

this is nonbreaking addition of CTA/CTD